components/link/stories/link.stories.js

```diff
@@ -89,105 +98,81 @@ Default.tags = ["!autodocs"];
 // ********* DOCS ONLY ********* //
 /**
  * The primary link is the default variant and is blue. This should be used to call attention to the link and for when the blue color won’t feel too overwhelming in the experience.
+ * All links can have a quiet style, without an underline. This style should only be used when the placement and context of the link is explicit enough that a visible underline isn’t necessary.
  */
-export const Primary = TemplateWithFillerText.bind({});
-Primary.args = {
-	...Default.args,
-	text: "link using spectrum-Link"
-};
+export const Primary = LinkGroupText.bind({});
+Primary.args = {};
 Primary.tags = ["!dev"];
 Primary.parameters = {
 	chromatic: { disableSnapshot: true },
 };
+Primary.storyName = "Standalone, primary";
 
 /**
  * The secondary variant is the same gray color as the paragraph text. Its subdued appearance is optimal for when the primary variant is too overwhelming, such as in blocks of text with several references linked throughout.
+ * Link is using `spectrum-Link--secondary`
```

Comment:
Can we make this statement and the ones you have above the static colors stories in the comments into sentences? And maybe throw in the word "class" somewhere like "This link is using the `spectrum-Link--secondary` class." or "Link is using the class `spectrum-Link--secondary`." I think it makes it a little clearer and it's also searchable in case someone is looking for information on classes on the page.